src/services/item/plugins/publication/publicationState/index.test.ts

```diff
@@ -0,0 +1,83 @@
+import { StatusCodes } from 'http-status-codes';
+import { v4 } from 'uuid';
+
+import { FastifyInstance } from 'fastify';
+
+import { HttpMethod } from '@graasp/sdk';
+
+import build, { clearDatabase } from '../../../../../../test/app';
+import { ITEMS_ROUTE_PREFIX } from '../../../../../utils/config';
+import { saveMember } from '../../../../member/test/fixtures/members';
+import { ItemTestUtils } from '../../../test/fixtures/items';
+
+const testUtils = new ItemTestUtils();
+
+// mock datasource
+jest.mock('../../../../../plugins/datasource');
+
+describe('Publication Controller', () => {
+  let app: FastifyInstance;
+  let actor;
```

Comment:
This variable is `any`, it can create unexpected behavior.

```suggestion
  let actor: Actor;
```

Reply:
Good catch !